code/backend/batch/utilities/search/SearchHandlerBase.py

```diff
@@ -31,9 +33,5 @@ def delete_files(self, files, id_field):
         pass
 
     @abstractmethod
-    def query_search(self, question):
-        pass
-
-    @abstractmethod
-    def return_answer_source_documents(self, search_results):
```

Comment:
Any reason why this is being removed as opposed to being renamed to `_convert_to_source_documents`?